addons/ballistics/CfgAmmo.hpp

```diff
@@ -6,6 +6,88 @@ class CfgAmmo {
         timeToLive=6;
     };
 
+    class B_12Gauge_Pellets_Submunition : BulletBase { //#00 Buckshot
```

Comment:
Delete the space before the :

Reply:
Half of this file does ` : ` and the other half does `: ` should I change all of them?